src/flags/orgFlags.ts

```diff
@@ -10,7 +10,14 @@ import { Messages, Org, ConfigAggregator } from '@salesforce/core';
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/sf-plugins-core', 'messages');
 
-const maybeGetOrg = async (input?: string): Promise<Org | undefined> => Org.create({ aliasOrUsername: input });
+const maybeGetOrg = async (input?: string): Promise<Org | undefined> => {
```

Comment:
My use case for this flag is to specify a prod org, which I believe is another category of of a well known org. How does one bind a different config value to the flag?